meson_options.txt

```diff
@@ -2,9 +2,9 @@ option('enable-test', type: 'boolean', value: true)
 option('enable-orc', type: 'boolean', value: true) # default true, use orc when found orc library
 option('install-test', type: 'boolean', value: false)
 option('enable-tensorflow-lite', type: 'boolean', value: true)
-option('enable-tensorflow', type: 'boolean', value: true)
-option('enable-caffe2', type: 'boolean', value: true)
-option('enable-pytorch', type: 'boolean', value: true)
+option('enable-tensorflow', type: 'boolean', value: false)
+option('enable-caffe2', type: 'boolean', value: false)
+option('enable-pytorch', type: 'boolean', value: false)
```

Comment:
How about disabling other sub-plugins (e.g., `tensorflow-lite`) as well in `meson_options.txt`?
I think enabling them by default doesn't seem like a good approach due to forced-dependency.
A build script of each system may enable them if required.

Reply:
@dongju-chae You made a good point and we need to discuss it. However, those options are disabled by mistake (for the test purpose). At least, in this PR, I will set them to true:)